packages/core/admin/ee/server/services/user.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+
+const { pipe, castArray, map, toNumber } = require('lodash/fp');
+
+/** Checks if ee disabled users list needs to be updated
+ * @param {string} id
+ * @param {object} input
+ */
+const shouldUpdateEEDisabledUsersList = async (id, input) => {
+  const data = await strapi.db.query('strapi::ee-store').findOne({
+    where: { key: 'ee_disabled_users' },
+  });
```

Comment:
Not agains of using a store, but why not having a separate table for storing disabled users?
Don't we store the 'isActive' field in the original user table already?

Reply:
What happens if a user is removed? Do we plan to remove the orphan id in this list?  Pbbly there will not be a lot of disabled users per project, so maybe it's not a problem.

Reply:
We need a separate list that is not going to be transferable on data import/export. And this is a copy so that we can track who we disabled. If we change it directly in the user table and then admin user changes it again, we have no record or knowledge that we have ever disabled the user

Reply:
isn't the ee-store transfered over the data import/export?

Reply:
Also, was trying to follow the path of where we update the user. But I thought it was through the admin.

Where do you actually update if the user is active or not?

Reply:
but the store might contain more info than only the disabled users I imagine, we might not want to disable the transfer for everything no? @Convly Or is it possible eto filter a specific row of a table to not be transfered?

Reply:
Are you looking for `packages/core/admin/server/services/user.js` `updateById` ?

Reply:
And even if that row is not transfered, the User `isActive` field will be transfered. 

Don't you expect, if the admin user `isActive` is false, to actually be disabled?

Reply:
> Are you looking for packages/core/admin/server/services/user.js updateById ?

Opps, I mean from the Admin UI , the update controller, where is it called? When updating the user entity?